client/tests/unit/App.unit.test.tsx

```diff
@@ -1,38 +1,11 @@
 import React from 'react';
-import { render, screen, act } from '@utils/testUtils';
+import { render, screen } from '@utils/testUtils';
 import App from '@/App';
-import { vi } from 'vitest';
-import useWatchAuthenticatedState from '@hooks/useWatchAuthenticatedState';
-
```

Comment:
I've been trying to find the best way to test our App's Routing and Authentication with our current layout in my draft PR.  We should try to sync on this at some point this week if you want